message/html/compose.js

```diff
@@ -64,6 +75,12 @@ exports.create = function (api) {
     })
     textArea.publish = publish // TODO: fix - clunky api for the keyboard shortcut to target
 
+    // load draft
+    let draft = api.drafts.sync.get(resolve(meta).root || '/public')
```

Comment:
The `where` logic is repeated, could move it up, and call it something like `draftLocation`